### DISCUSSION TITLE
[CHANGE] on web documents

### DISCUSSION THREAD

**Discussion by mgifford** [gc-da11yn/gc-da11yn.github.io#243] [Standard on Information and Communication Technology (ICT) Accessibility / Norme]
2.3.3.1 Departments must provide an accessible version of any non-web documents when requested.
https://a11y.canada.ca/en/standards/standard/#2.3.3.1

This needs to clearly be listed as an exception. You just can't allow non-web documents to be delivered that aren't accessible, unless it is exceptional. Lots of non-web documents can be made more accessible.  

Specifically with PDFs, the government needs to stop the bleeding. Every new PDF is future technical debt. Yes, you can technically make them accessible, but this is hard and limited in many ways. It is just too easy to produce inaccessible PDFs and too hard to determine if they are reasonably accessible. 

PDFs are encapsulated paper documents and are not digital native. https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/

**Comment by bruce-usab** [comment]
That article is new-to-me, thanks Mike!

Strong +1 (from painful experience) that phrasing similar to "accessible versions provided upon request" is a red flag, especially for anything recent and/or public facing.  As a checkbox option while registering for an in-person event, fine.  As a proactive attempt at indemnification?  It may have the opposite effect!

Accessibility of born digital content in U.S. .gov space is also a work in progress.

**Comment by mmiron-HC** [comment]
@mgifford should the subject line read non-web?

**Comment by skdonnelly** [comment]
The linked PDF https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf does list many WCAG requirements for non-web content such as PDFs, why are we not applying the same standards? As mentioned above the "on request" option pretty much opens the door to people posting their content in non-accessible PDF format. IMO it should say something like:
**2.3.3 Non-Web documents**
2.3.3.1 Any non-web documents posted to the web or available for download must conform with the latest version of [EN 301 549 (PDF, 2.3MB)] clause 10 and its sub-clauses. All other non-web documents must be available in an accessible version on request. (https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf)
